script/tool/lib/src/gradle_check_command.dart

```diff
@@ -216,6 +216,31 @@ buildscript {
 apply plugin: "com.google.cloud.artifactregistry.gradle-plugin"
 ''';
 
+  /// String printed as a valid example of settings.gradle repository
+  /// configuration that enables artifact hub env variable.
+  /// GP stands for the gradle plugin method of flutter tooling inclusion.
+  @visibleForTesting
+  static String exampleSettingsArtifactHubStringGP = '''
+// See $artifactHubDocumentationString for more info.
+plugins {
+    id "dev.flutter.flutter-plugin-loader" version "1.0.0"
+    // ...other plugins
+    id "com.google.cloud.artifactregistry.gradle-plugin" version "2.2.1"
+}
+  ''';
+
+  /// String printed as a valid example of settings.gradle repository
+  /// configuration without the artifact hub env variable.
+  /// GP stands for the gradle plugin method of flutter tooling inclusion.
+  @visibleForTesting
+  static String exampleSettingsWithoutArtifactHubStringGP = '''
```

Comment:
Why is this defined here? It only seems to be used in tests.

Reply:
Moved it to the test file.